tests/test_record_resource.py

```diff
@@ -90,19 +103,58 @@ def test_create_update_record(client, minimal_record):
     """Test record update."""
     # Create dummy record to test update
     response = client.post(
-        "/api/records_v2", headers=HEADERS, data=json.dumps(minimal_record)
+        "/records_v2", headers=HEADERS, data=json.dumps(minimal_record)
     )
     assert response.status_code == 200
     recid = response.json["pid"]
 
     # Update the record
     new_title = "updated title"
     minimal_record["titles"][0]["title"] = new_title
-    response = client.put("/api/records_v2/{}".format(recid), headers=HEADERS,
+    response = client.put("/records_v2/{}".format(recid), headers=HEADERS,
                           data=json.dumps(minimal_record))
     assert response.status_code == 200
 
     # Read the record
-    response = client.get("/api/records_v2/{}".format(recid), headers=HEADERS)
+    response = client.get("/records_v2/{}".format(recid), headers=HEADERS)
     assert response.status_code == 200
     assert response.json["metadata"]["titles"][0]["title"] == new_title
+
+
+def test_create_record_permissions(app_with_custom_permissions, client,
+                                   minimal_record, users):
+    """Test record creation."""
+    user1 = users['user1']
+    user2 = users['user2']
+    # Create new record as anonymous
+    with pytest.raises(PermissionDeniedError) as e:
+        response = client.post(
+            "/records_v2", headers=HEADERS, data=json.dumps(minimal_record)
+        )
```

Comment:
Normal/Discussion: This is where the topic of error handling comes in. My take is that we should test that the response is a serialized error i.e. `response.status_code == 403` and message is non-empty. This would make sure that we have a system in place for converting errors to http errors correctly. 

I would be fine with shelving that to another issue.

Reply:
I think this will bother me tomorrow while I am implementing the `tombstone` handling. I saw that the `PermissionDeniedError` isn't handled. I think we can go ahead with this one and I will address it in my next PR ;)